nautobot/utilities/management/commands/makemigrations.py

```diff
@@ -1,5 +1,5 @@
 # noinspection PyUnresolvedReferences
-from django.core.management.commands.makemigrations import Command
+from django.core.management.commands.makemigrations import Command  # noqa: F401
```

Comment:
Seems to be required 

```
Traceback (most recent call last):
  File "/usr/local/bin/nautobot-server", line 5, in <module>
    main()
  File "/source/nautobot/core/cli.py", line 61, in main
    initializer=_configure_settings,  # Called after defaults
  File "/source/nautobot/core/runner/runner.py", line 266, in run_app
    management.execute_from_command_line([runner_name, command] + command_args)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 244, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 38, in load_command_class
    return module.Command()
AttributeError: module 'nautobot.utilities.management.commands.makemigrations' has no attribute 'Command'
```

Reply:
Ah, good to know. In that case, adding it to `__all__` would make it clear that this isn't used locally but is provided for external consumption, i.e.:

```python
__all__ = (
    "Command",
)
```

Reply:
Look good now?